data/json/itemgroups/Food/food.json

```diff
@@ -183,6 +183,7 @@
       { "item": "can_chowder", "prob": 35 },
       { "item": "can_herring", "prob": 30 },
       { "item": "can_chicken", "prob": 40 },
+      { "item": "bread_canned", "prob": 25 },
```

Comment:
Is canned bread really this common?

Reply:
I'm not sure. I thought 25 was fair, unless you suggest otherwise.

Reply:
I think it would be better in the fours or so, but I am unsure how common it is in New England.

Reply:
https://sanelinews.com/remember-canned-bread-its-making-a-comeback-and-selling-like-hotcakes/

I mean it pops up in all major grocery stores in NE. I have a feeling this should be moved to a different item group...

Reply:
brown bread id very popular in new england but canned bread is not